parlai/tasks/wikipedia/agents.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
```

Comment:
i think some comment about which wikipedia dump here also would help.
also what the different variants do (i.e. copy from tast_list.py)